components/logstash/scripts/configure.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+from os.path import (
+    dirname,
+    join,
+)
+
+from cloudify import ctx
+
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+import utils  # NOQA
+
+
+CONFIG_PATH = 'components/logstash/config'
+LOGSTASH_SERVICE_NAME = 'logstash'
+
+
+def configure_logstash():
+
+    logstash_conf_path = '/etc/logstash/conf.d'
+
+    rabbitmq_username = ctx.instance.runtime_properties['rabbitmq_username']
```

Comment:
Good catch. Since there's a conditional below that checks that both `rabbitmq_username` and `rabbitmq_password` are properly set, it makes sense to use `.get` to get a nice error description instead of a key error.

I'll add a new comment once I've verified and pushed the change.